daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/IdeLedgerClient.scala

```diff
@@ -46,7 +46,10 @@ class IdeLedgerClient(
 
   def currentSubmission: Option[ScenarioRunner.CurrentSubmission] = _currentSubmission
 
-  private[this] val preprocessor = new engine.preprocessing.CommandPreprocessor(compiledPackages)
+  private[this] val preprocessor =
+    // TODO: https://github.com/digital-asset/daml/pull/10504
```

Comment:
If we support it without suffixes in sandbox classic I’m not sure we gain anything by avoiding it in the scenario service.

Reply:
comment dropped.